src/stores/settings.ts

```diff
@@ -49,6 +50,10 @@ export const useSettingsStore = defineStore('settings', {
     },
     userSatisfactionPollData: {},
 
+    ActivityData: {
+      neverTreatAsAfkFilter: '',
+    },
```

Comment:
I'd rather you put this directly in the top-level of the settings store instead of wrapping in `ActivityData`.